Formula/gnirehtet.rb

```diff
@@ -0,0 +1,36 @@
+class Gnirehtet < Formula
+  desc "Reverse tethering tool for Android"
+  homepage "https://github.com/Genymobile/gnirehtet"
+  url "https://github.com/Genymobile/gnirehtet/releases/download/v2.3/gnirehtet-java-v2.3.zip"
+  sha256 "93d1d46ee566376596f033832626dd5e89e76c91f2c46d2383735937b7d3b8b0"
+
+  depends_on :java => "1.8+"
```

Comment:
Quoting upstream website:

> Two implementations of Gnirehtet are available: one in Java; one in Rust.
> Which one to choose?
> Use the Rust implementation.

Reply:
I would have preferred the Rust version too but there 2.3 Rust version for Mac isn't up, just Windows and Linux.

Would you recommend sticking with the 2.3 Java version or the 2.2.1 Rust version?

Reply:
We build formulas from source, and the sources for 2.3 are published, so that will work :)

Reply:
Done.